File: python-example/tests/data_providers.py
import time
from model.customer import Customer


def current_time_millis():
    return int(round(time.time() * 1000))

valid_customers = [Customer(firstname="Adam", lastname="Smith", phone="+0123456789",
                            address="Hidden Place", postcode="12345", city="New City",
                            country="US", zone="KS",
                            email="adam%s@smith.me" % current_time_millis(),
                            password="qwerty"),
                   # ...
                   ]

File: csharp-example/csharp-example/tests/DataProviders.cs
using System;
using System.Collections;

namespace csharp_example
{
    internal class DataProviders
    {
        public static IEnumerable ValidCustomers
        {
            get
            {
                yield return new Customer()
                {
                    Firstname = "Adam",
                    Lastname = "Smith",
                    Phone = "+0123456789",
                    Address = "Hidden Place",
                    Postcode = "12345",
                    City = "New City",
                    Country = "US",
                    Zone = "KS",
                    Email = "adam" + (long)(DateTime.Now - DateTime.MinValue).TotalMilliseconds + "@smith.me",
                    Password = "qwerty"
                };
                /* ... */
            }
        }
    }
}